webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -98,6 +104,7 @@ export class ContactsEffects {
   }
 
   private loadContact(id) {
+    const trackPerformance = this.performanceService.track('select_contact:load_everything:contact_data');
```

Comment:
How about 'select_contact:load_everything:load_contact_data' for consistency ?

Reply:
Ah makes sense! thanks